include/OpenColorIO/OpenColorIO.h

```diff
@@ -3366,13 +3394,22 @@ class OCIOEXPORT GpuShaderDesc : public GpuShaderCreator
 
     // 1D lut related methods
     virtual unsigned getNumTextures() const noexcept = 0;
+    OCIO_DEPRECATED("This was marked as deprecated starting in v2.3, please use getTexture() with dimensions.")
+    virtual void getTexture(unsigned /*index*/,
+                            const char *& /*textureName*/,
+                            const char *& /*samplerName*/ ,
+                            unsigned & /*width*/,
+                            unsigned & /*height*/,
+                            TextureType & /*channel*/,
+                            Interpolation & /*interpolation*/) const {}
```

Comment:
Why having an empty default implementation? 
The implementation could redirect to the new method and hide the `dimensions` attribute. By doing so, the implementation remains correct.